Smarter handling for transition area QC metrics

labkey-jeckels opened the pull request:
#### Rationale
Transition-level data isn't always quantitative. When it's not, it doesn't make sense to use it as a QC metric. Also, users want to be able to visualize precursor and transition peak areas independently, or together

#### Changes
* Enable transition-level area metrics only when we have quantitative data
* Expose transition and precursor areas as their own top-level metrics so they can get the full Levey-Jennings treatment
* Rename the "Peak Area" metric to make it clear that it's the combination of precursor and transition areas

Commit:
Smarter handling for transition area QC metrics

ankurjuneja commented on resources/queries/targetedms/QCMetricEnabled_transitionArea.sql:
```diff
@@ -0,0 +1,45 @@
+    WHERE
+        (Quantitative = TRUE) OR (Quantitative IS NULL AND
+                                  (FragmentType != 'precursor' AND
+                                  t.GeneralPrecursorId.GeneralMoleculeId.PeptideGroupId.RunId IN (
```
is this indentation intentional?

labkey-jeckels replied:
Kind of. IntelliJ has strange ideas for how to indent this SQL and I got tired of fighting it.

Commit:
Make tests happier

Improve formatting for instrument serial number/model in QC summary

Commit:
Avoid double-counting metrics that are plotted alone and paired

Delete unused code

Fix links in QC summary table to propagate metricId into URL
